requirements-docs.txt

```diff
@@ -17,7 +17,7 @@ sphinx
 sphinx-rtd-theme
 nose
 Cython
-mock
+unittest
```

Comment:
I'm a little concerned about this being here.  Since `unittest` is part of the python standard library, I get this when I try to `pip install unittest`:
```
ERROR: Could not find a version that satisfies the requirement unittest (from versions: none)
ERROR: No matching distribution found for unittest
```

I'm concerned that merging this would cause our sphinx autodoc build to fail.

Reply:
oops!